### ISSUE TITLE
Add structured-log support

### ISSUE THREAD

**Issue by robertpmansion** [ngx-rocket/generator-ngx-rocket#409]
## I'm submitting a...

- [ ] Bug report  
- [x] Feature request
- [ ] Documentation issue or request

## Current behavior
You can debug the app by going to the developer console.

## Expected behavior
It would be great if this can be integrated with serilog.

## Environment

```
- generator version: 5.3.0
- node version: 9.5.0
- npm version: 5.6.0
- OS:  Windows 10 Pro x64
```

Others:

**Comment by bursauxa** [comment]
While it is an interesting topic, it is very much specific to only one server technology. I am unsure whether this would exceed the scope of ngx-rocket. Any thoughts @sinedied ?

**Comment by sinedied** [comment]
Indeed, specific solutions like these could/should be implement by userland add-ons, especially as the current provided logger can be extended easily with other outputs, see https://github.com/ngx-rocket/generator-ngx-rocket/issues/154
